Docker/README.md

```diff
@@ -87,15 +93,15 @@ With the additional parameter provided, the resulting command will look as follo
 
 - For local execution with the results in the UI:
      ```
-        docker run -it -p 8080:8080 \
```

Comment:
@sepich please review this line.

Reply:
This command (`--show-report`) runs container which would never exit. 
We need `-it` here to be able to stop serving of UI via `Ctrl-C`

Reply:
even with `-it`, `Ctrl-c` doesn't work for me. It does for you?

Reply:
Yes, but it is broken in this exact container version. It will work in next one.

Reply:
added back the -it then.